### DISCUSSION TITLE
Can't access my account after changing my email.

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#6553] [Firefly III]
Hello everyone. I am trying to log back into my account after changing my email. I never received a email asking me to confirm the email change. More importantly, I never configured the email part. When i go to sign in with my new email, I get a message saying " Until you verify your email address, you cannot login." 

Is there anyway around this? Can I somehow a.reset the old email or b.export my data? I looked up the documentation where it mentions  [command line usage](https://docs.firefly-iii.org/firefly-iii/pages-and-features/commandline/). However I did not notice anything about a password/username reset/change. I did find a command that mentions exports `firefly-iii:export-data` . But when I run this, I get this 
 `local.ERROR: Invalid access token for user #1`  
`local.ERROR: Token given is "", expected something else`.  
`Invalid access token. Check /profile`

Is my data lost?

**Comment by JC5** [comment]
Check out the search. It's also in the documentation. It happens to more people.

**Reply by ghost** [reply]
I looked up some past discussion regarding this issue and found one from February of this year where you mention the solution. I'm not to familiar with mariadb. [Re-send verification email #5811](https://github.com/firefly-iii/firefly-iii/discussions/5811#discussioncomment-2236193url). Can you help me out with this? Or point to me to some KBA where I can change the DB
